assets/video-section.css

```diff
@@ -1,6 +1,7 @@
 .video-section__media {
+  --ratio-percent: 56.25%;
   position: relative;
-  padding-bottom: 56.25%;
+  padding-bottom: calc(var(--ratio-percent) - var(--media-border-width));
```

Comment:
I think the support in Safari is too recent: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/14-32-uomhq-fh0q4.png)
</details>

I think we support the last 2 versions not including the current one so 14 & 15 but I might be wrong on this. It might be 15 and 16.

Reply:
It has been too recent. Maybe something we can discuss with the team now. We'd want to see what the worst case scenario is for browsers without support as part of that discussion/investigation. Out of scope for this PR though.